core/storage/question/gae_models_test.py

```diff
@@ -27,17 +27,34 @@
 class QuestionModelUnitTests(test_utils.GenericTestBase):
     """Tests the QuestionModel class."""
 
-    def test_create_question(self):
+    def test_create_question_empty_skill_id_list(self):
         state = state_domain.State.create_default_state('ABC')
         question_state_data = state.to_dict()
         language_code = 'en'
         version = 1
         question_model = question_models.QuestionModel.create(
-            question_state_data, language_code, version)
+            question_state_data, language_code, version, [])
 
         self.assertEqual(
             question_model.question_state_data, question_state_data)
         self.assertEqual(question_model.language_code, language_code)
+        self.assertItemsEqual(question_model.linked_skill_ids, [])
+
+    def test_create_question_with_skill_ids(self):
+        state = state_domain.State.create_default_state('ABC')
+        question_state_data = state.to_dict()
+        linked_skill_ids = ['skill_id1', 'skill_id2']
+        language_code = 'en'
+        version = 1
+        question_model = question_models.QuestionModel.create(
+            question_state_data, language_code, version,
+            linked_skill_ids)
+
+        self.assertEqual(
+            question_model.question_state_data, question_state_data)
+        self.assertEqual(question_model.language_code, language_code)
+        self.assertItemsEqual(
+            question_model.linked_skill_ids, linked_skill_ids)
```

Comment:
Ditto. linked_skill_ids is a list.

Reply:
Oh I see you are using assertItemsEqual. This test method will not be available for python 3, so it might be better to use self.assertEqual(question_model.linked_skill_ids, set(linked_skill_ids))

Reply:
Python 3 has assertCountEqual to do the same https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual